LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Inpsyde GmbH
+Copyright (c) 2023 Inpsyde GmbH
```

Comment:
maybe we should change it to "Since 2006" to avoid changing this string every year 😂

Reply:
Unfortunately this is a legal requirement" and we can't do whatever we like... _In theory_ we should add the copyright with the year in ever file. But it has  become a common practice to no add the year in ever every file, but just "see License" and then add the year in the license file. But the year must be updated manually every year. I was thinking on a PHPCS rule to check the year of license...